AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/backgroundreplacement/BackgroundReplacementConfiguration.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  BackgroundReplacementConfiguration.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import UIKit
+
+@objcMembers public class BackgroundReplacementConfiguration {
+    var backgroundReplacementImage: UIImage
```

Comment:
use `let` instead of `var` if those variables are not going to change.

Reply:
done